src/main/java/racingcar/view/Input.java

```diff
@@ -0,0 +1,65 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Input {
+    public String requestCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String carNames = Console.readLine();
+
+        validateFormat(carNames);
+        validateString(carNames);
+        validateUnique(carNames);
+        return carNames;
+    }
+
+    public String requestRaceRounds() {
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        String raceRounds = Console.readLine();
+
+        validateFormat(raceRounds);
+        validateInteger(raceRounds);
+        return raceRounds;
+    }
+
+    private void validateFormat(String input) {
+        if (input == null || input.isEmpty() || input.equals(" ")) {
+            throw new IllegalArgumentException("잘못된 입력입니다. 널, 개행, 공백 문자가 아닌 문자열을 입력해주세요.");
+        }
+    }
+
+    private void validateString(String input) {
+        if (input.matches(".*[0-9].*")) {
+            throw new IllegalArgumentException("잘못된 입력입니다. 숫자가 포함되지 않은 문자열을 입력해주세요.");
+        }
+    }
```

Comment:
예외 조건을 메서드로 분리해서 이름으로 의도를 드러내면 더 좋을 것 같습니다 !

```suggestion
    private void validateString(String input) {
        if (containsNumber(input)) {
            throw new IllegalArgumentException("잘못된 입력입니다. 숫자가 포함되지 않은 문자열을 입력해주세요.");
        }
    }
```

Reply:
오 확실히 가독성이 향상되었네요! 피드백 감사합니다!